Written about comoda results (#21)
thesis20/report: sections/experiments/results.tex

\subsection{Results of experiments}
In this section we look at the results from the experiments.
\subsubsection{LDOS-CoMoDa}
For the experiments conducted on the LDOS-CoMoDa dataset, we should see the methods that handle context do well, as this dataset has many context variables that the rating depends on.

> As mentioned, we are looking at to of these context variables, which are the \textit{dominantEmo} and the \textit{physical} attributes as these are the most influential.

Reviewer: ```suggestion
As mentioned, we are looking at two of these context variables, which are the \textit{dominantEmo} and the \textit{physical} attributes as these are the most influential.
```
according to?

The methods that handle context are \texttt{Itemsplitting} and the \texttt{CAMF} methods.

> We see that \texttt{Itemsplitting} has the largest Recall, but also the smallest Precision@10, resulting in a small F1@10 value, and in general, it does not perform well on LDOS-CoMoDa, which is surprising as it is able to utilize the context.

Reviewer: ```suggestion
We see that \texttt{Itemsplitting} has the largest recall@10, but also the smallest precision@10, resulting in a small F1@10 value, and in general, it does not perform well on LDOS-CoMoDa, which is surprising as it is able to utilize the context.
```

> However, \texttt{Itemsplitting} is able to achieve one of the highest NDCG's.

Reviewer: ```suggestion
However, \texttt{Itemsplitting} is able to achieve one of the highest NDCG scores.
```

Both \texttt{CAMF-C} and \texttt{CAMF-CI} perform well on LDOS-CoMoDA in terms of most metrics, except for MAP@K.
They are able to achieve the lowest prediction errors amongst all methods in terms of RMSE and MAE, which could indicate that they are able to make good use of the context.
Close to \texttt{CAMF} methods we have baselines such as \texttt{SVD} and \texttt{SVD++} which both are competitive in terms of Precision@10, Recall@10 and F1@10. 

> These also achieve the highest MAP@K values but do not have as low prediction errors as the \texttt{CAMF} methods.

Reviewer: ```suggestion
These also achieve the highest MAP@K values but do not have as low prediction errors as the \texttt{CAMF} methods.
It should, however, be noted that these methods were created with a focus on MAE and RMSE scores, and not precision or recall.
```

Reviewer: måske med en kilde til netflix price bedømmelse ting

\texttt{NMF} generally perform a little worse in all metrics compared to \texttt{SVD}, except for NDCG.
The \texttt{DeepWalk + kNN} method has high prediction errors and low Precision@10 and MAP@10 values but has the second-highest Recall@10 and one of the highest NDCG value.

> Finally, we have the methods that do not handle rating predictions but only do recommendations, so these do not have RMSE and MAE values.

Reviewer: ```suggestion
Finally, we have the methods that do not handle rating predictions but only do top K recommendations, so these do not have RMSE and MAE values, since they work with interactions rather than ratings.
```

We see that \texttt{LightGCN} is not able to perform well on any of the metrics.

> We suspect that this is because the LDOS-CoMoDa dataset is as small as it is with only 121 users, 1232 unique movies, and 2296 ratings, and it is therefore not able to utilize its strengths.

Reviewer: ```suggestion
We suspect that this is because the LDOS-CoMoDa dataset is as small as it is with only 121 users, 1232 unique movies, and 2296 ratings, and it is therefore not able to utilize its strengths.
And all of these numbers are considerably lower on the actual data used, due to pruning.
```

The \texttt{DeepWalk + kNN} method has high prediction errors and low Precision@10 and MAP@10 values but has the second-highest Recall@10 and one of the highest NDCG value.
Finally, we have the methods that do not handle rating predictions but only do top K recommendations, so these do not have RMSE and MAE values, since they work with interactions rather than ratings.
We see that \texttt{LightGCN} is not able to perform well on any of the metrics.
We suspect that this is because the LDOS-CoMoDa dataset is as small as it is with only 121 users, 1232 unique movies, and 2296 ratings, and it is therefore not able to utilize its strengths.
And all of these numbers are considerably lower on the actual data used, due to pruning.
However, \texttt{NGCF} is able to achieve both the highest Precision@10 as well as the highest NDCG but does have a lower Recall@10 resulting in a low F1@10 score.
\\\\
To summarize, we expected the context-aware methods to perform well on the LDOS-CoMoDA dataset, as the ratings are context-dependent. 
The \texttt{CAMF} methods do perform well as expected, but \texttt{Itemsplitting} is underperforming in most metrics.

> Some baselines such as \texttt{SVD} and \texttt{SVD++} closely comptetive with \texttt{CAMF} as the best performing methods.

Reviewer 2: ```suggestion
Some baselines such as \texttt{SVD} and \texttt{SVD++} are closely competitive with \texttt{CAMF} as the best performing methods.
```
